pages/gallery.html

```diff
@@ -14,16 +14,58 @@
     <title>Galeria - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
-    </nav>
     <section class="small-hero small-hero__image--gallery ">
       <div class="small-hero__layer"></div>
       <div class="small-hero__layer small-hero__layer--darker"></div>
       <h1 class="small-hero__title">Galería</h1>
     </section>
+    <nav>
+      <!-- nav bar section -->
+    </nav>
+    <!-- gallery -->
+    <section class="container container-componet gallery">
+      <div class="gallery__info">    
+        <p class="text-p">Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia. Observa todo lo que tenemos para ti en el majestuoso paraíso de Capurganá</p>
+      </div>      
+      <div class="gallery__items" id="divImagesContainer" >
+        <div class="gallery-item"><div class="gallery-item__images"></div>
+        <div class="gallery-item__overlay">
+          <img src="./../assets/images/gallery/expand_icon.svg" class="overlay__image" alt=""></div>
+        </div>
+        <div class="gallery-item"><div class="gallery-item__images"></div>
+        <div class="gallery-item__overlay">
+          <img src="./../assets/images/gallery/expand_icon.svg" class="overlay__image" alt=""></div>
```

Comment:
can you give an enter between `<img src="./../assets/images/gallery/expand_icon.svg" class="overlay__image" alt="">` and `</div>`